src/library/components/Select.tsx

```diff
@@ -0,0 +1,199 @@
+import {
+  ChangeEvent,
+  FocusEvent,
+  SelectHTMLAttributes,
+  useCallback,
+  useEffect,
+  useRef,
+  useState,
+  createContext,
+  MutableRefObject,
+  forwardRef,
+  ForwardedRef
+} from 'react';
+import { valueMissingError } from '../helpers/validation';
+
+export const SelectOptionAddonPropsContext = createContext({
+  ariaInvalid: false,
+  ariaDescribedby: '',
+  errorText: '',
+  isDirty: false,
+  setIsDirty: (isDirty: boolean) => {
+    return;
+  },
+  setFieldsetError: (text: string) => {
+    return;
+  }
+});
```

Comment:
I don't think we are using an of these in SelectOption. Can you confirm and remove if true?